homeassistant/components/home_connect/api.py

```diff
@@ -347,14 +369,14 @@ class Washer(
 
     def get_entity_info(self):
         """Get a dictionary with infos about the associated entities."""
-        door_entity = self.get_door_entity()
+        door_entities = self.get_door_entities()
```

Comment:
Similar variable elimination comments as the others.

Reply:
Same comment as above, changing this feels out of scope of this PR.